plans/kn6254_thv92.md

```diff
@@ -0,0 +1,61 @@
+* Kim Yu Ng kn6254
+* Thien Vo thv92
+
+### Yo Dawg, Don't Just Watch 
+
+[$Scope.$Watch](http://cdn.memegenerator.net/instances/500x/43555515.jpg) - AngularJS
+
+
+## Status
+
+* We have our development environment set up and currently looking for UI changes.
+
+* We have updated dependencies in bower.json
+
+* We have changed from leaflet to openlayers javascript library which is used to load, display and render maps from multiple sources on web pages.
+
+* We are looking how to integrate javascript or jquery into AngularJS.
+
+```javascript
+/* The left and right buttons can then be use to navigate through the pages of the card */
+
+$(document).keydown(function(e) {
+    switch(e.which) {
+        case 37: // left
+        //make call to angular function
+        break;
+
+        case 39: // right
+        //make call to angular function
+        break;
+
+        default: return; // exit this handler for other keys
+    }
+    e.preventDefault();
+});
+```
+
+
+* We are looking for UI enhancement and an intuitive design of Card on the page.
+
+
+## Goal
+
+* We want to find out the reason and fix the issue of slow zooming in or zooming out on open layers when loading actual layer from geoserver.
```

Comment:
I believe openlayers is not any slower than leaflet. We're trying to port from leaflet to openlayers. However, we don't have deep understanding on how the technology works. So there might be some issue on our current porting